client/src/reducers/__tests__/items-entities-reducer.test.js

```diff
@@ -0,0 +1,57 @@
+import itemsEntities from "../items-entities-reducer";
+import { getTodo } from "../../actions/get-all-todos-action";
+import { addAnotherItem } from "../../actions/add-todo-action";
+import { deleteTodo } from "../../actions/delete-todo-action";
+import { updateStatus } from "../../actions/toggle-todo-action";
+
+test("should return the initial state", () => {
+	expect(itemsEntities(undefined, { type: undefined })).toEqual({ todos: [] });
+});
+
+test("should handle a todo being added to an empty list", async () => {
+	const previousState = [];
+	expect(
+		itemsEntities(previousState, {
+			type: "ADD_TODO_SUCESS",
+			todos: "Run the tests",
```

Comment:
this is a good test but as a best practice you should prepare the item added as variable in the start of the test instead of just hard code it in the action